superset-frontend/src/SqlLab/components/QuerySearch.jsx

```diff
@@ -39,6 +39,15 @@ const propTypes = {
   displayLimit: PropTypes.number.isRequired,
 };
 
+const TableStyles = styled.div`
+  height: ${props => props.height};
```

Comment:
Not sure if it's too much to tackle this in this PR (we can create an Issue/ticket if it is), but I would LOVE to stop passing heights/widths around as props (not just here, but all over the codebase), and start using things like flexbox layouts to tackle these issues. It would reduce a lot of rendering and increase performance.

Reply:
Done. That was much easier than I expected and we got rid of a lot of code in SqlLab/components/App